### PR TITLE
Proposed 0.5.4 version

### PR DESCRIPTION
**Pull request by mfalt**
- Updated version
- Added Julia 1.2 to test
- Fixed really weird bug in norminf

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@ language: julia
 julia:
   - 1.0
   - 1.1
```
Comment (baggepinnen) [inline .travis.yml]:
```suggestion
```
Looks good, can stop testing on 1.1 if we test on 1.2

**Comment by mfalt** [comment]
We have a few problems:
- I don't know how to reliably sort the eigenvalues so that we get complex conjugated poles together, which is needed for some methods (fix_conjugate_pairs)
- Acker does not return a real K even when the poles are complex-conjugated.

**Comment by baggepinnen** [comment]
Take complex conjugate of all poles with negative imaginary part and then
solve the assignment problem using the Hungarian algorithm?

**Comment by mfalt** [comment]
> Take complex conjugate of all poles with negative imaginary part and then solve the assignment problem using the Hungarian algorithm?

This seems like it would require us to put some epsilon for when they should be matched or not. It is certainly a possibility, but seems a bit unnecessary when solvers always solve the conjugates together. I created an alternative PR with "no sorting"

**Pull request merged**